### PR TITLE
Smithing with pugilisitic intent

### PR DESCRIPTION
**Pull request by MicroMinty**
Adds knuckledusters as smithables!

## About The Pull Request
Adds knuckledusters to the smithables pool, with a recipe of Draw, Pull, Pull, Bend

## Why It's Good For The Game
Another item to have fun with as a smith, and more access to viable pugilistic weapons.

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog

:cl:
add: smithable knuckledusters
/:cl:

### CODE REVIEW

**Comment by Leon9621** [comment]
29 or so damage with 8 quality and titanium, also attacking three times a second? H o o  b o y

File: code/modules/smithing/smithed_items.dm [hunk 1]
```diff
@@ -636,5 +636,17 @@
...
+	finalitem = /obj/item/melee/smith/knuckleduster
 
-
+/obj/item/smithing/knuckleduster/startfinish()
+	var/obj/item/melee/smith/knuckleduster/finalforreal = new /obj/item/melee/smith/knuckleduster(src)
+	finalforreal.force += quality*1.1
+	finalforreal.throwforce = finalforreal.force*1.1
+	finalforreal.attack_speed = CLICK_CD_MELEE * (3 / quality)
+	finalitem = finalforreal
+	..()
```
Comment (ProbablyCarl) [inline code/modules/smithing/smithed_items.dm:652]: No newline.

**Comment by MicroMinty** [comment]
I'm not entirely sure why the failing check is failing, or what it does. Please assist?

**Comment by MicroMinty** [comment]
> > I'm not entirely sure why the failing check is failing, or what it does. Please assist?
> 
> Read the comment. No newline. Just add an empty line at the end of the file.

which file? I'm a bit out of it due to irl circumstances and idk if im just missing it or...

**Comment by ProbablyCarl** [comment]
[code/modules/smithing/smithed_items.dm](https://github.com/sunset-wasteland/sunset-wasteland/pull/617/files/1bd1afdbf497deb36a31fdf98812349516d5fc8b#diff-3e847afd63638a4c28e2e9961b4ca458f26dddfb660fb3af411f26dfcdbcdce1)

**Comment by ProbablyCarl** [comment]
Assuming this is dead. DM me on Discord if you want this re-open.

**Comment by ProbablyCarl** [comment]
Re-opened as per author request.

**Comment by MicroMinty** [comment]
> 29 or so damage with 8 quality and titanium, also attacking three times a second? H o o b o y

You know? I forgot to factor for materials. Oops!

**Comment by MicroMinty** [comment]
I'll be lowering damage and fixing the newline thing, give me a day or so.

**Comment by Leon9621** [comment]
> > 29 or so damage with 8 quality and titanium, also attacking three times a second? H o o b o y
> 
> You know? I forgot to factor for materials. Oops!

To be fair, titanium only gives it 10% or so damage. It's more the attack speed, HOO boy that's gonna be fun to fuck around with.

**Comment by MicroMinty** [comment]
> 

I nerfed the attack power anyway, but yeah the point is the attack speed. funny punch go brr and all

**Comment by MicroMinty** [comment]
agony. missing newlines in smithed items still. I wonder where?

**Comment by MicroMinty** [comment]
> Please test the weapon in game first on your local host against various target of various armors....

I had, and it seemed fine to me, however that was only done with iron, so I should probably go back and do it with titanium....   well, there's reason for this to be marked as do not merge anyway already

**Comment by ProbablyCarl** [comment]
Missing new lines on smithing.dm

**Comment by MicroMinty** [comment]
> Missing new lines on smithing.dm

I know, I have gone in and fixed what I thought was the only missing newline.

**Comment by MicroMinty** [comment]
do not merge yet even if it works, am re-doing sprites

**Comment by MicroMinty** [comment]
sprites redone

**Comment by MicroMinty** [comment]
woah, what the fuck.

**Review by Moribun — APPROVED** [review]

**Review by Moribun — COMMENTED** [review]
I honestly think it seems pretty solid. bice to see new smithed items

**Pull request closed without merging**